components/logins/src/logins.udl

```diff
@@ -69,6 +69,18 @@ dictionary Login {
 /// These are the errors returned by our public API.
 [Error]
 interface LoginsApiError {
+    /// NSS not initialized.
+    NSSUninitialized();
+
+    /// NSS error during authentication
+    NSSAuthenticationError(string reason);
+
+    /// error during authentication (in PrimaryPasswordAuthenticator)
+    AuthenticationError(string reason);
+
+    /// authentication has been cancelled.
+    AuthenticationCancelled();
```

Comment:
fwiw, `Canceled` appears more common in american engine, so maybe change this. OTOH, neither you or I are American, so 🤷 :)